Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
```

Comment:
My only preference is maybe use python3 here ... Avoids another manual thing we’ll forget to upgrade ... Sorry if I missed a reason for pinning 3.8

Reply:
What about just `python:latest`? Or do we want to hold back when `4.0` comes out? :sweat_smile: